gammapy/scripts/spectrum_pipe.py

```diff
@@ -0,0 +1,86 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+from ..spectrum import (
+    SpectrumEnergyGroupMaker,
+    FluxPointEstimator,
+    SpectrumExtraction,
+    SpectrumFit,
+    SpectrumResult,
+)
+from ..background import ReflectedRegionsBackgroundEstimator
+
+__all__ = [
+    'SpectrumAnalysisIACT',
+]
+
+log = logging.getLogger(__name__)
+
+
+class SpectrumAnalysisIACT(object):
+    """High-level analysis class to perform a full 1D IACT spectral analysis
+
+    Observation selected must have happend before. For a usage example see
```

Comment:
"selected" -> "selection"
Start "For a usage example" on a new line (maybe even separated by a blank line).